t/001-management.t

```diff
@@ -0,0 +1,29 @@
+use Test::Nginx::Socket::Lua 'no_plan';
+use Cwd qw(cwd);
+
+my $pwd = cwd();
+
+our $HttpConfig = qq{
+    lua_package_path "$pwd/src/?.lua;;";
+};
+
+our $management = qq{
+    include $pwd/conf.d/management.conf;
+};
+
+repeat_each(2);
+no_root_location();
+run_tests();
+
+__DATA__
+
+=== TEST 1: management
+This is just a simple demonstration of the
+echo directive provided by ngx_http_echo_module.
+--- http_config eval: $::HttpConfig
```

Comment:
Some info on the eval in the book: https://openresty.gitbooks.io/programming-openresty/content/testing/preparing-tests.html
Or in the documentation http://search.cpan.org/dist/Test-Nginx/lib/Test/Nginx/Socket.pm#Filters

It just uses that variable.